banners/desktop/content/BannerText.vue

```diff
@@ -10,7 +10,7 @@
 				heutigen {{ currentDayName }}, den {{ currentDate }}, bitten wir Sie bescheiden, die Unabhängigkeit
 				von Wikipedia zu unterstützen. {{ campaignDaySentence }} <AnimatedText :content="visitorsVsDonorsSentence"/> Die meisten
 				Menschen spenden, weil sie Wikipedia nützlich finden. Die durchschnittliche Spende beträgt 22,25&nbsp;€,
-				doch bereits 5&nbsp;€ helfen uns weiter. Hat Wikipedia Ihnen in diesem Jahr Wissen im Wert einer Tasse
+				doch bereits 5 € helfen uns weiter. Hat Wikipedia Ihnen in diesem Jahr Wissen im Wert einer Tasse
```

Comment:
not sure if that `&nbsp;` flew out here by accident or if it was part of the font problem solving

Reply:
Good catch, it was by accident. PhpStorm now hides the `&nbsp;` character making it hard to tell if there's one in there or not..